src/Entity/Developer.php

```diff
@@ -223,7 +250,58 @@ public function hasApp(string $appName): bool {
    * {@inheritdoc}
    */
   public function getCompanies(): array {
-    return $this->decorated->getCompanies();
+    // If companies is null it means the original API response that this
```

Comment:
@peterserfozo Could you please write a test for this method to ensure it always works as it is described?